vendor/composer/ClassLoader.php

```diff
@@ -313,29 +334,34 @@ public function loadClass($class)
      */
     public function findFile($class)
     {
-        // work around for PHP 5.3.0 - 5.3.2 https://bugs.php.net/50731
-        if ('\\' == $class[0]) {
-            $class = substr($class, 1);
-        }
```

Comment:
No, it’s not a breaking change. According to the comment, it’s a workaround for an issue in PHP `5.3.0`-`5.3.2`, so I’d guess that the issue has been fixed in `5.3.3`. The last PHP `5.3.x` release was `5.3.29`.